darwin/future/core/workflows/get_workflows.py

```diff
@@ -0,0 +1,13 @@
+from typing import List, Optional
+
+from pydantic import parse_obj_as
+
+from darwin.future.core.client import Client
+from darwin.future.data_objects.workflow import Workflow
+
+
+def get_workflows(client: Client, team_slug: Optional[str] = None) -> List[Workflow]:
+    team_slug = team_slug or client.config.default_team
+    response = client.get(f"/v2/teams/{team_slug}/workflows?worker=false")
+
+    return [parse_obj_as(Workflow, workflow) for workflow in response]
```

Comment:
If one of these fails, it'll fail to return the entire list. Which kind of breaks a bit from what we've been doing recently with the Tuple(List[Object], List[Exception]).

Reply:
We could maybe write a generic that solves this
```python
def expand_exception_parse(response, T) -> Tuple[T, Exception]:
    ...
```